src/stores/useMessageStore.js

```diff
@@ -25,46 +23,40 @@ export default function useMessageStore() {
   const keyBlocklist = ['isSet', 'labels', 'equivalent_identifiers', 'type', 'id', 'degree', 'synonyms'];
   let unknownNodes = false;
 
-  function graphToIndMap(graph, type) {
+  function toIndMap(arr) {
```

Comment:
What is ind? Index? I think you should either use the full name, or add some kind of documentation explaining what this is.